popups/scratch-messaging/popup.html

```diff
@@ -68,7 +68,12 @@
           </div>
         </bdo>
         <div class="reply-box-comment" v-show="replying">
-          <textarea class="reply-textarea" maxlength="500" v-model="replyBoxValue"></textarea>
+          <textarea
+            class="reply-textarea"
+            maxlength="500"
+            v-model="replyBoxValue"
+            @keyup.enter="postComment($event.shiftKey)"
```

Comment:
Does look ugly a bit IMO, also I'm pretty sure you broke the button

Reply:
> Does look ugly a bit IMO

Blame vue1 not supporting `.ctrl` modifier

Reply:
Can't you `&&` in the HTML?

Reply:
that would look uglier but sure